com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -17,11 +17,7 @@
 
 namespace Unity.Robotics.UrdfImporter
```

Comment:
For future reference, we will be wanting to update the namespaces as per Robotics -> Simulation namespace change

Reply:
Thanks. But out of the scope of this PR.